Analytics/Classes/Internal/SEGUserDefaultsStorage.h

```diff
@@ -0,0 +1,20 @@
+//
+//  SEGUserDefaultsStorage.h
+//  Analytics
+//
+//  Created by Tony Xiao on 8/24/16.
+//  Copyright © 2016 Segment. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SEGStorage.h"
+
+@interface SEGUserDefaultsStorage : NSObject <SEGStorage>
+
+@property (nonatomic, strong, nullable) id<SEGCrypto> crypto;
```

Comment:
@tonyxiao Possibly, given support has been very recently added (just since the last release) I'm not too worried about migrating it cleanly for tvOS, if it helps us implementing this.

But as I said, doesn't need to be a blocker, should be ok as is and we can refine it later since the FileStorage class is internal.